src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -3762,74 +3761,86 @@ def test_reprod_view(capsys, monkeypatch, annexed, dist_mem):
     result, _ = capsys.readouterr()
     if dist_mem:  # annexed can be True or False
         expected = (
-            isched + "[invoke='invoke_0', dm=True]\n"
-            "    " + directive+"[OMP parallel]\n"
-            "        " + directive + "[OMP do][reprod=True]\n"
-            "            " + loop + "[type='dofs', "
+            isched + "[invoke='invoke_0', dm=True]\n" +
+            indent + "0: " + directive + "[OMP parallel]\n" +
+            2*indent + sched + "[]\n" +
```

Comment:
I like the `x*indent`. Much more readable and methodical than the comparisons with long strings of spaces :)